benchmarking/smartseq2/merge_expression_matrix/run_merge_quants.wdl

```diff
@@ -0,0 +1,38 @@
+task mergeTPM{
```

Comment:
do we want to keep the same composability structure for these workflows as well?  Move all of the tasks out to their own wdls and have the workflows import?